peps/api/index.rst

```diff
@@ -25,67 +26,80 @@ The structure is like:
        "requires": string | null,
        "replaces": string | null,
        "superseded_by": string | null,
+       "author_list": Array<string>,
        "url": string
      },
    }
 
 Date values are formatted as DD-MMM-YYYY,
 and multiple dates are combined in a comma-separated list.
 
-For example:
+A selection of example PEPs are shown here,
+illustrating some of the possible values for each field:
 
 .. code-block:: json
 
    {
-     "8": {
-       "number": 8,
-       "title": "Style Guide for Python Code",
-       "authors": "Guido van Rossum, Barry Warsaw, Alyssa Coghlan",
+     "12": {
```

Comment:
Why change these examples?

@JelleZijlstra suggested these examples in https://github.com/python/peps/pull/3864#discussion_r1677217465:

> Suggestion: use 8, 484 (two of the most well-known PEP numbers; 484 has a non-empty `topic`), and 622 (demonstrates superseded_by). 

3124 (replacing 622) does not have `superseded_by`.

Reply:
PEP 8 gets too much attention! It was also to include a PEP with a comma in an author's name, and 3124 has 'Requires' (and is an anagram of 1234!).